httpobs/scanner/local.py

```diff
@@ -0,0 +1,71 @@
+import httpobs.conf
+
+from httpobs.scanner.analyzer import NUM_TESTS, tests
+from httpobs.scanner.grader import get_grade_and_likelihood_for_score
+from httpobs.scanner.retriever import retrieve_all
+
+
+def scan(hostname, **kwargs):
+    # Always allow localhost scans when run in this way
```

Comment:
Not sure but I think docstring extractors might break with the comment being first, maybe move it after the docstring?